integration-tests/bot/jail_test.py

```diff
@@ -61,14 +62,10 @@ def create_addresses(self):
     def unjail(self,name, passphrase, address):
         try:
             return self.rpc.staking.unjail(address, name)
-        except KeyboardInterrupt as ex:
-            print("unjail fail={}".format(ex))
-        except SystemExit as ex:
-            print("unjail fail={}".format(ex))
-        except Exception as ex:
+        except jsonrpcclient.exceptions.JsonRpcClientError as ex:
             print("unjail fail={}".format(ex))
         except :
```

Comment:
Why not simply remove this except clause, I think it still will cause the warning.